src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -522,13 +524,26 @@ export function startAppointmentCancel(appointment) {
 export function confirmCancelAppointment() {
   return async (dispatch, getState) => {
     const appointment = getState().appointments.appointmentToCancel;
+    const featureVAOSServiceRequests = selectFeatureVAOSServiceRequests(
+      getState(),
+    );
+    const featureVAOSServiceVAAppointments = selectFeatureVAOSServiceVAAppointments(
+      getState(),
+    );
 
     try {
       dispatch({
         type: CANCEL_APPOINTMENT_CONFIRMED,
       });
 
-      const updatedAppointment = await cancelAppointment({ appointment });
+      const updatedAppointment = await cancelAppointment({
+        appointment,
+        useV2:
```

Comment:
@cferris32 I changed this logic when I was re-reviewing our changes. The way I wrote it initially, we would have sent all cancellations through the V2 endpoints if `featureVAOSServiceRequests` was on. Now it checks the appropriate feature toggles for requests vs confirmed appointments.